pkg/sabakan/main.go

```diff
@@ -178,6 +195,24 @@ func subMain(ctx context.Context) error {
 	}
 	s.ListenAndServe()
 
+	// HTTPS API
+	webServerHTTPS := web.NewServer(model, cfg.IPXEPath, cryptsetupPath, advertiseURL, advertiseURLHTTPS, allowedIPs, cfg.Playground, counter, true)
+	ss := &well.HTTPServer{
+		Server: &http.Server{
+			Addr:    cfg.ListenHTTPS,
+			Handler: webServerHTTPS,
+		},
+		ShutdownTimeout: 3 * time.Minute,
+		Env:             env,
+	}
+	err = ss.ListenAndServeTLS(cfg.ServerCertFile, cfg.ServerKeyFile)
+	if err != nil {
+		if !errors.Is(err, fs.ErrNotExist) {
+			return err
+		}
+		fmt.Fprintln(os.Stderr, "TLS certificate not found.  Starting without serving TLS API server.")
```

Comment:
Why do you continue to run the sabakan server without HTTPS API?
In my opinion, if required files are missing, we should terminate the program.

Reply:
I implemented this by the following reason.
https://github.com/cybozu-go/sabakan/pull/270#discussion_r1363019197

But in our case, we don't need this feature anymore. So I'll make the program to terminate when cert files are missing.